library/src/main/java/com/goddb/internal/DBImpl.java

```diff
@@ -159,6 +159,47 @@ public void del(String path, String condition) throws GoddbException {
             }
         }
     }
+    // ***********************
+    // *      UPDATE
+    // ***********************
+
+    @Override
+    public void update(String path, String condition, String modData) throws GoddbException {
+        boolean chk = false;
+        String[] section = modData.split(",");
+
+        ArrayList<Integer> wildcardArrayList = Wildcard.extractWildcard(path, mappingTable);
+
+        for (int curPath : wildcardArrayList) {
+            try {
+                JSONArray oldArray = new JSONArray(__get(String.valueOf(curPath)));
+                JSONArray conditionArray, retArray = new JSONArray();
+
+                conditionArray = Condition.extractCondition(oldArray, condition);
+
+                for (int i = 0; i < oldArray.length(); i++) {
+                    for (int j = 0; j < conditionArray.length(); j++) {
+                        if (oldArray.getJSONObject(i).toString().equals(conditionArray.getJSONObject(j).toString())) {
+                            chk = true;
+                        }
+                    }
+                    if (!chk) {
+                        for(int k = 0; k < section.length ; k++) {
+                            String[] data = section[k].split(":");
+                            oldArray.getJSONObject(i).put(data[0], data[1]);
+                        }
+                        retArray.put(oldArray.getJSONObject(i));
+                    }
+                    chk = false;
+                }
```

Comment:
finally, delete the original path's object and then put the conditionArray to the database for replacing that.